submission/persistence-common-api/src/main/kotlin/ac/uk/ebi/biostd/persistence/common/service/SubmissionOperations.kt

```diff
@@ -55,14 +47,38 @@ interface SubmissionPersistenceQueryService {
      * @param filter the submission filter
      **/
     fun getSubmissionsByUser(owner: String, filter: SubmissionFilter): List<BasicSubmission>
+}
+
+interface SubmissionRequestPersistenceService {
+    fun hasActiveRequest(accNo: String): Boolean
+
+    fun saveSubmissionRequest(rqt: SubmissionRequest): Pair<String, Int>
+
+    fun createSubmissionRequest(rqt: SubmissionRequest): Pair<String, Int>
+
+    fun updateRequestStatus(accNo: String, version: Int, status: RequestStatus)
 
     fun getPendingRequest(accNo: String, version: Int): SubmissionRequest
+
     fun getLoadedRequest(accNo: String, version: Int): SubmissionRequest
+
     fun getCleanedRequest(accNo: String, version: Int): SubmissionRequest
-    fun getReferencedFiles(accNo: String, fileListName: String): List<ExtFile>
+
     fun getRequestStatus(accNo: String, version: Int): RequestStatus
 }
 
+interface SubmissionFilesPersistenceService {
+    fun saveSubmissionFile(file: SubmissionFile)
+
+    fun getSubmissionFile(path: String, accNo: String, version: Int): ExtFile
+
+    fun getSubmissionFiles(accNo: String, version: Int, startingAt: Int = 0): List<Pair<ExtFile, Int>>
```

Comment:
I think this kind of default parameters are dangerous, I mean IMO we should use default parameters for parameters that are optional, imagine for example operation submit optioanally may accept a on behalf user paraneter but here startAt is not optional is good that every class using the api actually provide a value and using default parameter may introduce a bug because of this.